submissions/daudaLawal/database/init.sql

```diff
@@ -0,0 +1,81 @@
+CREATE DATABASE food_bag;
+-- Creation of Database
+
+SHOW DATABASE;
+-- To show the list of all Databases
+
+USE food_bag;
+-- The USE key word helps to use food_bag for our task databse
+
+
+-- NOW TO TABLE CREATION
+
+-- Food Table Creation
+CREATE TABLE food (
+id INT NOT NULL AUTO_INCREMENT,
+type VARCHAR(100) NOT NULL,
+PRIMARY KEY (id)
+);
+
+-- SHOW TABLE
+    SHOW TABLES;
+
+-- Admins Table Creation
+CREATE TABLE Admins (
+id INT NOT NULL AUTO_INCREMENT,
+name VARCHAR(100) NOT NULL,
+type ENUM ('0','1') NOT NULL,
+email_address VARCHAR(255) NOT NULL,
+phone_number VARCHAR(50) NOT NULL,
+password VARCHAR(100) NOT NULL,
+PRIMARY KEY (id)
+);
+
+
+-- SHOW TABLE
+SHOW TABLES;
+
+-- Vendor table creation
+
+CREATE TABLE vendor_food (
+id INT NOT NULL AUTO_INCREMENT,
+admin_id INT NOT NULL,
+food_id INT NOT NULL,
+name VARCHAR(100) NOT NULL,
+amount DECIMAL(20,2) NOT NULL,
+PRIMARY KEY (id),
+FOREIGN KEY (food_id) REFERENCES food (id),
+FOREIGN KEY (admin_id) REFERENCES Admins (id)
+);
+
+
+-- SHOW TABLE
+    SHOW TABLES;
+
+-- Customers table creation
+
+CREATE TABLE Customers (
+id INT NOT NULL AUTO_INCREMENT,
+name VARCHAR(100) NOT NULL,
+phone_number VARCHAR(50) NOT NULL,
+address VARCHAR(250) NOT NULL,
+payment_method ENUM('Card','Mobile Money','Cash'),
```

Comment:
add this to line 62: NOT NULL DEFAULT 'Card' . This is because payment_method, should be as seen on the payment verification page. In addition, the first four columns mentioned are required and the other columns regarding payment are optional.